From 3ff1d732c6bcd9add69957d34704e2eba5d04cd4 Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Tue, 16 Apr 2019 19:17:23 +0300
Subject: [PATCH 1095/1200] arm64: dts: cn9131: move network nodes to board
 level dts

This patch is preparation to SetupB creation.
Move network nodes and serdes description to board level dts and efuse
from dts to dtsi.

Change-Id: Icda913a9ee8af5a12312e9f412484b824cf3b5d3
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/7724
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Kostya Porotchkin <Kostya.Porotchkin@cavium.com>
---
 arch/arm/dts/cn9131-db-A.dts | 38 ++++++++++++++++++++++++++++++++++-
 arch/arm/dts/cn9131-db.dtsi  | 39 +-----------------------------------
 2 files changed, 38 insertions(+), 39 deletions(-)

diff --git a/arch/arm/dts/cn9131-db-A.dts b/arch/arm/dts/cn9131-db-A.dts
index 132a56efe0..74a19c7dc2 100644
--- a/arch/arm/dts/cn9131-db-A.dts
+++ b/arch/arm/dts/cn9131-db-A.dts
@@ -14,7 +14,43 @@
 		     "marvell,armada-ap806";
 };
 
-&cp1_ld_efuse1 {
+&cp1_comphy {
+	/* Serdes Configuration:
+	 *	Lane 0: PCIe0 (x2)
+	 *	Lane 1: PCIe0 (x2)
+	 *	Lane 2: unconnected
+	 *	Lane 3: USB1
+	 *	Lane 4: SFP (port 0)
+	 *	Lane 5: SATA1
+	 */
+	phy0 {
+		phy-type = <COMPHY_TYPE_PEX0>;
+	};
+	phy1 {
+		phy-type = <COMPHY_TYPE_PEX0>;
+	};
+	phy2 {
+		phy-type = <COMPHY_TYPE_UNCONNECTED>;
+	};
+	phy3 {
+		phy-type = <COMPHY_TYPE_USB3_HOST1>;
+	};
+	phy4 {
+		phy-type = <COMPHY_TYPE_SFI0>;
+		phy-speed = <COMPHY_SPEED_10_3125G>;
+	};
+	phy5 {
+		phy-type = <COMPHY_TYPE_SATA1>;
+	};
+};
+
+&cp1_ethernet {
 	status = "okay";
 };
 
+/* CON50 */
+&cp1_eth0 {
+	status = "okay";
+	phy-mode = "sfi"; /* lane-4 */
+	marvell,sfp-tx-disable-gpio = <&cp1_gpio0 9 GPIO_ACTIVE_HIGH>;
+};
diff --git a/arch/arm/dts/cn9131-db.dtsi b/arch/arm/dts/cn9131-db.dtsi
index 198229019a..7b3be011ad 100644
--- a/arch/arm/dts/cn9131-db.dtsi
+++ b/arch/arm/dts/cn9131-db.dtsi
@@ -60,45 +60,8 @@
 	};
 };
 
-&cp1_comphy {
-	/* Serdes Configuration:
-	 *	Lane 0: PCIe0 (x2)
-	 *	Lane 1: PCIe0 (x2)
-	 *	Lane 2: temporary unconnected
-	 *	Lane 3: USB1
-	 *	Lane 4: SFP (port 0)
-	 *	Lane 5: SATA1
-	 */
-	phy0 {
-		phy-type = <COMPHY_TYPE_PEX0>;
-	};
-	phy1 {
-		phy-type = <COMPHY_TYPE_PEX0>;
-	};
-	phy2 {
-		phy-type = <COMPHY_TYPE_UNCONNECTED>;
-	};
-	phy3 {
-		phy-type = <COMPHY_TYPE_USB3_HOST1>;
-	};
-	phy4 {
-		phy-type = <COMPHY_TYPE_SFI0>;
-		phy-speed = <COMPHY_SPEED_10_3125G>;
-	};
-	phy5 {
-		phy-type = <COMPHY_TYPE_SATA1>;
-	};
-};
-
-&cp1_ethernet {
-	status = "okay";
-};
-
-/* CON50 */
-&cp1_eth0 {
+&cp1_ld_efuse1 {
 	status = "okay";
-	phy-mode = "sfi"; /* lane-4 */
-	marvell,sfp-tx-disable-gpio = <&cp1_gpio0 9 GPIO_ACTIVE_HIGH>;
 };
 
 &cp1_i2c0 {
-- 
2.22.0

